cypress/README.md

```diff
@@ -0,0 +1,13 @@
+# Cypress Tests
```

Comment:
Can we please add this information to the README we have in the root directory? And then remove this file?

And for folder structure, I'd like to keep it simple and have one folder called `e2e` in the root directory and that folder just has one file in it `goals.cy.ts`.